hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -206,4 +209,82 @@
       tests: ["keymgr_stress_all"]
     }
   ]
+  covergroups: [
+    {
+      name: state_and_op_cg
+      desc: '''
+            - Cover all operations with cdi_sel, dest_sel and op_status (only fail or success)
+            at any of all working_states.
+            - This is sampled once an operation is done.'''
+    }
+    {
+      name: lc_disable_cg
+      desc: '''
+             - Cover LC disable occurs at any of all the states or during any of all the
+             operations.
+             - This is sampled once LC disables keymgr.'''
+    }
+    {
+      name: sideload_clear_cg
+      desc: '''
+            - Cover all the `sideload_clear` values are used after any of all the operations and in
+            any of all the states.
+            - Cover `sideload_clear` with any combination of availability of 3 sideload interfaces.
+            - This is sampled once sideload_clear is programmed after an operation.'''
+    }
+    {
+      name: reseed_interval_cg
+      desc: '''
+            - Cover small values of reseed_interval are used, so that TB can actually check EDN
+              request is sent in the right interval.
+            - Also Cover some large values to ensure all bits are toggled'''
```

Comment:
- Also c(lower case C)over some large values to ensure all bits are toggled. --> add a period?